core/server/master/src/main/java/alluxio/master/metastore/InodeStore.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.metastore;
+
+import alluxio.master.file.meta.Inode;
+import alluxio.master.file.meta.InodeView;
+
+/**
+ * Inode metadata storage.
+ *
+ * The inode store manages metadata about individual inodes, as well as the parent-child
+ * relationships between them.
+ */
+public interface InodeStore extends ReadOnlyInodeStore {
```

Comment:
for these APIs, would it make sense to have the parameters be `InodeView`, instead of `long id`? I feel like it would make more sense as the user of this interface to deal with `InodeView` and not always get the `id`.

Reply:
makes sense, done